app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml

```diff
@@ -61,32 +60,50 @@
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toBottomOf="@+id/admin_pin_main_text" />
 
-        <org.oppia.android.app.profile.ProfileInputView
-          android:id="@+id/input_pin"
+        <com.google.android.material.textfield.TextInputLayout
+          android:id="@+id/admin_pin_input_pin"
+          style="@style/TextInputLayout"
           android:layout_width="0dp"
           android:layout_height="wrap_content"
           android:layout_marginTop="48dp"
-          app:inputLength="5"
-          app:isPasswordInput="true"
-          app:label="@string/admin_pin_new_pin"
+          app:errorMessage="@{viewModel.pinErrorMsg}"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@+id/admin_pin_warning_text"
-          profile:error="@{viewModel.pinErrorMsg}" />
+          app:layout_constraintTop_toBottomOf="@+id/admin_pin_warning_text">
 
-        <org.oppia.android.app.profile.ProfileInputView
-          android:id="@+id/input_confirm_pin"
+          <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/admin_pin_input_pin_edit_text"
+            style="@style/TextInputEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/admin_pin_new_pin"
+            android:imeOptions="actionNext"
+            android:inputType="numberPassword"
+            android:maxLength="5" />
+        </com.google.android.material.textfield.TextInputLayout>
+
+        <com.google.android.material.textfield.TextInputLayout
+          android:id="@+id/admin_pin_input_confirm_pin"
+          style="@style/TextInputLayout"
           android:layout_width="0dp"
           android:layout_height="wrap_content"
           android:layout_marginTop="16dp"
-          app:inputLength="5"
-          app:isPasswordInput="true"
-          app:label="@string/admin_pin_new_confirm"
+          app:errorMessage="@{viewModel.confirmPinErrorMsg}"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintHorizontal_bias="0.0"
           app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@+id/input_pin"
-          profile:error="@{viewModel.confirmPinErrorMsg}" />
+          app:layout_constraintTop_toBottomOf="@+id/admin_pin_input_pin">
+
+          <com.google.android.material.textfield.TextInputEditText
```

Comment:
Add this to support RTL for TextInputLayout
`android:textDirection="locale"`

Reply:
done